### PR TITLE
reactUtil RSC Tests - RSC-1409

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-1409

### CODE REVIEW

File: lib/src/util/__tests__/reactUtil.test.tsx [hunk 1]
```diff
@@ -118,22 +161,35 @@ describe('reactUtil', function() {
     });
 
     it('passes through elements as-is', function() {
```
Comment (entroform) [inline lib/src/util/__tests__/reactUtil.test.tsx:163]: Isn't this the same as line 26?

File: lib/src/util/__tests__/reactUtil.test.tsx [hunk 2]
```diff
@@ -143,66 +199,98 @@ describe('reactUtil', function() {
     });
 
     it('wraps arrays that start with a string', function() {
-      const arr = ['bar', <span key="baz">baz</span>];
+      const arr = ['bar', <span key="baz">baz</span>],
+          result = ensureStartEndElements(arr) as ReactElement,
```
Comment (entroform) [inline lib/src/util/__tests__/reactUtil.test.tsx:203]: Should this be `ensureElement`?
Same as a few below

Comment (rpokorny) [reply lib/src/util/__tests__/reactUtil.test.tsx]: > Should this be ensureElement?
Same as a few below

Definitely.  Good catch. 423a4d257

**Review by entroform — APPROVED** [review]
LGTM!

**Pull request merged**